### PR TITLE
feat(web-analytics): Add live stats logic

### PR DESCRIPTION
**Pull request by robbie-c**
## Problem

The live users count service can show currently online users - and we're not doing anything with that :)

## Changes

Add a count for how many users are live in the project.

In page header:

Screenshot 2024-06-25 at 19 21 52

~~Normal view:~~ (placement is no longer here, it's in the header)

Screenshot 2024-06-25 at 16 36 34

~~Tooltip:~~ (the tooltip still looks like this, but the placement is now in the header)

Screenshot 2024-06-25 at 16 36 51

## Does this work well for both Cloud and self-hosted?

It's currently behind a feature flag, I'm not sure we support running this service in live-hosting yet

## How did you test this code?
Currently can't test this locally as I can't run the service locally, so had to mock the request. It's behind a FF so we'll dogfood before releasing

### CODE REVIEW

File: frontend/src/scenes/web-analytics/WebAnalyticsLiveUserCount.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import { teamLogic } from 'scenes/teamLogic'
+import { liveEventsTableLogic } from 'scenes/web-analytics/liveWebAnalyticsLogic'
+
+export const WebAnalyticsLiveUserCount = (): JSX.Element | null => {
+    const { liveUserCount, liveUserUpdatedSecondsAgo } = useValues(liveEventsTableLogic)
+    const { currentTeam } = useValues(teamLogic)
+
+    if (liveUserCount == null || liveUserUpdatedSecondsAgo == null) {
+        return null
+    }
```
Comment (Twixes) [inline frontend/src/scenes/web-analytics/WebAnalyticsLiveUserCount.tsx:14]: It's a bit odd that this stat completely disappears when nobody's active (and the disappearance also causes a large layout shift). I think this should always be present if the flag's on.

**Review by Twixes — COMMENTED** [review]
Love to see it. This was definitely one of the uses of this API that we had in mind.
I'm not sure about the placement in the UI – it feels somewhat accidental, not intentional. But it's alright.
Before we merge, one comment about the counter going AWOL

File: frontend/src/scenes/web-analytics/WebAnalyticsLiveUserCount.tsx (hunk 1 above)
Comment (robbie-c) [reply frontend/src/scenes/web-analytics/WebAnalyticsLiveUserCount.tsx]: Ah right! I didn't realise that the api returned undefined for 0 users, I'll fix it

Comment (robbie-c) [reply frontend/src/scenes/web-analytics/WebAnalyticsLiveUserCount.tsx]: Could add it as an action for the page like this:

Screenshot 2024-06-25 at 19 19 28

File: frontend/src/scenes/web-analytics/WebAnalyticsScene.tsx [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
 import { SceneExport } from 'scenes/sceneTypes'
+import { WebAnalyticsLiveUserCount } from 'scenes/web-analytics/WebAnalyticsLiveUserCount'
 import { webAnalyticsLogic } from 'scenes/web-analytics/webAnalyticsLogic'
 import { WebAnalyticsDashboard } from 'scenes/web-analytics/WebDashboard'
 
 export function WebAnalyticsScene(): JSX.Element {
-    return <WebAnalyticsDashboard />
+    return (
+        <>
+            <PageHeader buttons={<WebAnalyticsLiveUserCount />} />
```
Comment (Twixes) [inline frontend/src/scenes/web-analytics/WebAnalyticsScene.tsx:10]: This is pretty interesting, but IMO the nav should stay opinionated – the scene actions area is now a natural place to look for information like this, and putting this there dilutes the UX pattern of scene actions a bit. So I vote for putting it back in the scene area 😅

Comment (robbie-c) [reply frontend/src/scenes/web-analytics/WebAnalyticsScene.tsx]: Done!

**Review by Twixes — APPROVED** [review]

**Pull request merged**